.github/actions/getPullRequestDetails/getPullRequestDetails.js

```diff
@@ -9,7 +9,7 @@ const DEFAULT_PAYLOAD = {
 };
 
 const pullRequestNumber = ActionUtils.getJSONInput('PULL_REQUEST_NUMBER', {required: false}, null);
-const user = core.getInput('USER', {required: false});
+const user = core.getInput('USER', {required: true});
```

Comment:
Sorry for the delayed review here @HorusGoul, but I'm noticing that you made the `USER` input required here, but not in the `action.yml` metadata file.

Reply:
Done!

Reply:
It's a bit weird that we have to indicate it in two places. Shouldn't the action.yml file be enough?

Reply:
🤷 I don't make the rules 😆

Reply:
But lol, good point.